README.md

```diff
@@ -46,6 +46,21 @@ cd csharp\examples\simple
 dotnet run
 ```
 
+## Connecting to GCP GKE from local machine
+
+Start proxy to access kubernetes cluster:
+
+```bash
+$ kubectl proxy
```

Comment:
I don't think connecting through the proxy is the preferred way to sugest that people connect.